bundles/org.eclipse.passage.lbc.equinox/src/org/eclipse/passage/lbc/internal/equinox/ServerRunnerImpl.java

```diff
@@ -65,8 +66,21 @@ public void unbindLogger(LoggerFactory factory) {
 	public void bindServerHandler(BackendLauncher serverHandler, Map<String, Object> context) {
 		if (serverHandler != null) {
 			logger.info(NLS.bind(EquinoxMessages.ServerRunnerImpl_i_launcher_bind, serverHandler));
-			serverHandler.launch(context);
-			backendLaunchers.add(serverHandler);
+			LicensingResult result = serverHandler.launch(context);

```

Comment:
This was bad idea from my side to return result from the command, because this way we are going to wait for potentially long operation.
We need to create another ticket to rework it. It is better to use Consumer<LicensingResult> or BiConsumer if you need an instance of BackendLauncher.

Reply:
Will create a ticket to rework on Consumer